### ISSUE TITLE
Is there support for BatchNorm?

### ISSUE THREAD

**Issue by anirudth** [quic/aimet#51]
I get the following error `2020-05-21 16:16:35,240 - Winnow - ERROR - Unsupported op_type Mul, dotted net/Conv2d/BatchNorm/batchnorm_1/mul_1, input_ops: net/Conv2d/Conv2D]`

**Comment by quic-klhsieh** [comment]
Hi @anirudth, if your model is using non-fused BatchNormalization ops, support for this was added earlier today. Can you please retry with the latest code and see if you still encounter this issue?

Please note that we also only support BatchNorm ops created using tf slim or tf Keras apis.  Can you provide any additional details about the model you are using?

**Comment by anirudth** [comment]
Thanks for the quick reply. It seems to work.
